legend-engine-core/legend-engine-core-pure/legend-engine-pure-code-compiled-core/src/main/resources/core/pure/router/store/routing.pure

```diff
@@ -105,6 +112,42 @@ function <<access.private>> meta::pure::router::store::routing::getIncludedMappi
   $m->concatenate(if($m.includes->isEmpty(), |[], |$m.includes.included->map(i|$i->getIncludedMappingsRecursively())));
 }
 
+function meta::pure::router::store::routing::routeRelationFunctionSetImplementations(sets:SetImplementation[*], mapping:Mapping[1], runtime:Runtime[1], extensions: Extension[*]):SetImplementation[*]
+{
+  $sets->map(s|$s->routeRelationFunctionSetImplementation($mapping, $runtime, $extensions)); 
+}
+
+function meta::pure::router::store::routing::routeRelationFunctionSetImplementation(s:SetImplementation[1], mapping:Mapping[1], extensions: Extension[*]):SetImplementation[1]
+{
+  routeRelationFunctionSetImplementation($s, $mapping, ^Runtime(connectionStores=^ConnectionStore(connection=^Connection(),element='Mock')), $extensions);
+}
+
+function meta::pure::router::store::routing::routeRelationFunctionSetImplementation(s:SetImplementation[1], mapping:Mapping[1], runtime:Runtime[1], extensions: Extension[*]):SetImplementation[1]
+{
+   $s->match([
+              t:RelationFunctionInstanceSetImplementation[1]|^$t(relationFunction=$t->routeRelationFunction($mapping, $runtime, $extensions)),
+              s:InstanceSetImplementation[1]|$s,
+              o:OperationSetImplementation[1]|^$o(parameters = $o.parameters->map(i| ^$i(setImplementation=$mapping->classMappingById($i.id)->toOne()->routeRelationFunctionSetImplementation($mapping, $runtime, $extensions))))
+            ]);
+}
+
+function meta::pure::router::store::routing::routeRelationFunction(r:RelationFunctionInstanceSetImplementation[1], mapping:Mapping[1], runtime:Runtime[1], extensions: Extension[*]):FunctionDefinition<Any>[1]
+{
+  if($r.relationFunction->evaluateAndDeactivate().expressionSequence->at(0)->instanceOf(ClusteredValueSpecification),
+    | $r.relationFunction,
+    | $r.relationFunction->routeFunction(getRoutingStrategyFromMappingAndRuntime($mapping, $runtime), ^ExecutionContext(), [], $extensions, noDebug())->cast(@FunctionDefinition<Any>)
+  );
+}
+
+function meta::pure::router::store::routing::byPassClusteringInfo(value:Any[1]):Any[1]
```

Comment:
Is this function used?

Reply:
Yes, still need to bypass clustering info before passing to pure2sql flow - https://github.com/gs-rpant1729/legend-engine/blob/4bf3cf76254dd9ec64a1c790512d6bb7b6262d82/legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-generation/legend-engine-xt-relationalStore-pure/legend-engine-xt-relationalStore-core-pure/src/main/resources/core_relational/relational/pureToSQLQuery/pureToSQLQuery.pure#L3741